Symfony/app/Resources/views/base.html.twig

```diff
@@ -42,7 +42,15 @@
                         <a href="#contact">Contact</a>
                     </li>
                     <li>
-                        <a href="#login">Login</a>
+                    {% if is_granted('ROLE_USER') %}
+                        <a href="./logout">Logout</a>
+                    {% endif %}
+                        {% if not is_granted('ROLE_USER') %}
```

Comment:
You don't need to specify an `{% if %}` and then an `{% if not %}` for the same condition, use an `{% else %}`.